tests/foreman/api/test_location.py

```diff
@@ -300,3 +300,19 @@ def test_default_loc_id_check(self):
         """
         default_loc_id = entities.Location().search(query={'search': f'name="{DEFAULT_LOC}"'})[0].id
         assert default_loc_id == 2
+
+    @pytest.mark.tier1
+    def test_positive_get_location_by_name(self, make_entities):
+        """test to search location by name
+
+        :id: 9721bd0c-df51-11eb-9353-b0a460e02491
+
```

Comment:
I understand there is not much here, but could you write some `Steps` here?
Also for the other test?

Reply:
Added steps for this one but other test case does not require test steps as it is self explanatory. It will look redundant.